include/class.config.php

```diff
@@ -1277,6 +1281,7 @@ function updateTicketsSettings($vars, &$errors) {
         return $this->updateAll(array(
             'ticket_number_format'=>$vars['ticket_number_format'] ?: '######',
             'ticket_sequence_id'=>$vars['ticket_sequence_id'] ?: 0,
+            'ticket_counts'=>isset($vars['ticket_counts'])?1:0,
```

Comment:
I wouldn't call this `ticket_counts` - I think something like `queue_bucket_counts` would be more appropriate.